SUSYBSMAnalysis/HSCP/test/UsefulScripts/DeDxStudy/DeDxStudy.C

```diff
@@ -179,7 +182,11 @@ void DeDxStudy(string DIRNAME="COMPILE", string INPUT="dEdx.root", string OUTPUT
    gStyle->SetNdivisions(505,"X");
    TH1::AddDirectory(kTRUE);
 
-   bool isData = INPUT.find("MC")!=string::npos ? false : true;
+   bool isData   = !(INPUT.find("MC")      !=string::npos) ,
```

Comment:
But the MC files and the data files we are using are not in the Analysis_Samples.txt.